indigo/distribution.yaml

```diff
@@ -16417,6 +16417,31 @@ repositories:
       url: https://github.com/wavelab/ximea_camera.git
       version: devel-indigo
     status: maintained
+  xpp:
+    doc:
+      type: git
+      url: https://github.com/leggedrobotics/xpp.git
+      version: master
+    release:
+      packages:
+      - xpp
+      - xpp_examples
+      - xpp_hyq
+      - xpp_msgs
+      - xpp_quadrotor
+      - xpp_ros_conversions
+      - xpp_states
+      - xpp_vis
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/leggedrobotics/xpp-release.git
+      version: 1.0.0-1
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.

Reply:
@awinkler I have received you invitation. I will wait with accepting it until we have figured out why the Travis test didn't catch this automatically... @tfoote Can you please look into this?

Reply:
@dirk-thomas Ticketed at https://github.com/ros/rosdistro/issues/16181 please accept the request above we'll debug this later.

@ros/rosdistro-maintainers FYI the pull request checking isn't effective due to the above issue so we need to be on the lookout for it again.